src/philips_airctrl/coap/encryption.py

```
@@ -1,38 +1,41 @@
+    def __init__(self) -> None:
+        self._client_key: str | None = None
 
-    def set_client_key(self, client_key):
+    def set_client_key(self, client_key: str) -> None:
         self._client_key = client_key
 
-    def _increment_client_key(self):
-        client_key_int = (int(self._client_key, 16) + 1) & 0xFFFFFFFF  # Handle overflow
+    def _increment_client_key(self) -> None:
+        if self._client_key is None:
+            raise RuntimeError("Client key not initialised; call set_client_key() first")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)